content-src/components/DiscoveryStreamComponents/List/_List.scss

```diff
@@ -3,7 +3,12 @@ $item-font-size: 13;
 $item-line-height: 20;
 
 .ds-list-border {
-  border: 1px solid $grey-40-36;
+  border: 1px solid $grey-30;
+  border-left: 0;
+  border-right: 0;
+  border-bottom: 0;
```

Comment:
No need to reinvent the styling here with `$border-secondary` existing. Also you can reset all with just `border: 0`, so
```css
border: 0;
border-top: $border-secondary;
```

Reply:
good points; fixed!